src/app/base/components/SSHKeyList/SSHKeyList.test.tsx

```diff
@@ -216,59 +158,33 @@ describe("SSHKeyList", () => {
     });
   });
 
-  it("can delete a group of SSH keys", () => {
+  it("can delete a group of SSH keys", async () => {
     const store = mockStore(state);
-    const wrapper = mount(
-      <Provider store={store}>
-        <MemoryRouter
-          initialEntries={[
-            { pathname: "/account/prefs/ssh-keys", key: "testKey" },
-          ]}
-        >
-          <SSHKeyList />
-        </MemoryRouter>
-      </Provider>
-    );
+    renderWithBrowserRouter(<SSHKeyList />, {
+      route: "/account/prefs/ssh-keys",
+      store,
+    });
     // Click on the delete button:
-    wrapper
-      .find("tbody TableRow")
-      .at(1)
-      .findWhere((n) => n.name() === "Button" && n.text() === "Delete")
-      .simulate("click");
+    await userEvent.click(screen.getAllByText("Delete")[1]);
```

Comment:
Have you tried more specific query?
```suggestion
    await userEvent.click(screen.getAllByRole("button", { name: "Delete" })[1])
```